### PR TITLE
added an entry for getUTCSeconds term

### PR DESCRIPTION
**Pull request by rickc73**
### Description

- added an entry for getUTCSeconds
- Created getUTCSeconds.md
- Completed this task as part of the Git & GitHub course on CodeCademy
### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-developer-career-path'
+---
+
+The getUTCSeconds() method returns the seconds (ranging from 0 to 59) of a specific date according to UTC (Coordinated Universal Time).
```
Comment (KTom101) [inline content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md:15]:
```suggestion
The **`getUTCSeconds()`** method returns the seconds (ranging from 0 to 59) of a specific date according to UTC (Coordinated Universal Time).
```

File: content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Syntax
+
+
```
Comment (KTom101) [inline content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md:20]: Only need one line break here.
```suggestion
```

File: content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Date.getUTCSeconds()
+```
+
+A `Date` object is used for working with dates and times. The object provides a way to create, manipulate, and format dates in various formats.
+
+A `Date` object can be created by declaring a new variable with the `new Date()` constructor.
+
+## Example
+
+Consider the following example that uses the `.getUTCSeconds()` method to get the seconds of `2023-06-06T10:30:04` and logs the result to the console:
```
Comment (KTom101) [inline content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md:31]:
```suggestion
The following example uses the `.getUTCSeconds()` method to get the seconds of `2023-06-06T10:30:04` and logs the result to the console:
```

File: content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+console.log(seconds);
+```
+
+This example results in the following output:
+
+```shell
+4
+```
+The example below demonstrates the use of `.getUTCSeconds()` to print the current time in seconds to the console:
```
Comment (KTom101) [inline content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md:45]: Let's add a line break and make this a codebyte example.
```suggestion

## Codebyte Example

The codebyte example below uses the `.getUTCSeconds()` method to get the current time in seconds and logs it to the console:
```

File: content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+```js
```
Comment (KTom101) [inline content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md:47]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+const today = new Date();
+const today_seconds = today.getUTCSeconds();
+console.log(today_seconds) //ouput: current second of todays UTC time
```
Comment (KTom101) [inline content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md:50]:
```suggestion
console.log(today_seconds); 
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@rickc73 I've added a few changes to this entry.

**Review by rickc73 — COMMENTED** [review]
happy with the suggestions, i have applied them

File: content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-developer-career-path'
+---
+
+The **`getUTCSeconds()`** method returns the seconds (ranging from 0 to 59) of a specific date according to UTC (Coordinated Universal Time).
+
```
Comment (KTom101) [inline content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md:16]:
```suggestion
```

**Review by KTom101 — APPROVED** [review]
@rickc73 I noticed one more extra line break. Otherwise, it looks good for the next review, thanks! 😃

**Review by SSwiniarski — APPROVED** [review]
@rickc73 Looks good to me!

**Comment by SSwiniarski** [comment]
Congrats @rickc73 on the new entry. Live link here: https://www.codecademy.com/resources/docs/javascript/dates/getUTCSeconds

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**